docs/learn/expert-guides/universal-profile/extend-profile-functionalities.md

```diff
@@ -0,0 +1,556 @@
+---
+sidebar_label: '⚙️ Extend a Universal Profile Functionalities'
+sidebar_position: 4
+---
+
+# Extend a Universal Profile Functionalities
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its KeyManager to follow this guide. <br/>
+:arrow_left: If you don't have a Universal Profile yet, follow our previous guide [**Create a Universal Profile**](./create-profile.md).
```

Comment:
You might be able to remove the <br/> and just add one blank line - so to not mix HTML and markdown too much

```suggestion
You will need a Universal Profile that you can control via its KeyManager to follow this guide.

:arrow_left: If you don't have a Universal Profile yet, follow our previous guide [**Create a Universal Profile**](./create-profile.md).
```